src/KakaoMap.js

```diff
@@ -85,53 +43,33 @@ class KakaoMap extends React.Component {
                     center: new kakao.maps.LatLng(latitude, longitude),
                     level: level ? level : 3,
                 };
-                const map = new window.kakao.maps.Map(container, options);
-
-                dataList.forEach((data) => {
-                    const marker = new kakao.maps.Marker({
-                        map: map,
-                        position: new kakao.maps.LatLng(
-                            data.latitude,
-                            data.longitude
-                        ),
-                        image:
-                            markerImgSrc && markerImgWidth && markerImgHeight
-                                ? this.createMarkerImg(
-                                      markerImgSrc,
-                                      markerImgWidth,
-                                      markerImgHeight
-                                  )
-                                : '',
-                    });
-                    data.details
-                        ? this.createCustomOverlay(map, marker, data)
-                        : '';
-                });
+                let map = new window.kakao.maps.Map(container, options);
+                map.setZoomable(zoomable !== undefined ? zoomable : true);
+                this.setState({map});
             });
         };
     };
 
     render() {
-        return <Root id="map" />;
+        const { map } = this.state;
+        return (
+            <Root id="map">
+                {map ? (
```

Comment:
How about change this code to use && operator?

For example,
`{map && <Component />}`